src/devices/Interop/Windows/Kernel32.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    [DllImport("kernel32.dll")]
+    [return: MarshalAs(UnmanagedType.Bool)]
+    public static extern bool SetSystemTime([In] ref SystemTime lpSystemTime);
+
+    [DllImport("kernel32.dll")]
+    [return: MarshalAs(UnmanagedType.Bool)]
+    public static extern bool GetSystemTime([Out] out SystemTime lpSystemTime);
+
+    [Serializable]
```

Comment:
does this need to be serializable?

Reply:
Well, it's a flat struct, these should be made serializable by default.